src/tools/pontoon/search_filters.md

```diff
@@ -2,53 +2,56 @@
 
 <!-- toc -->
 
+## Search
+
+It’s possible to search within a project using the search field. Searches include strings, string IDs and comments.
```

Comment:
```suggestion
It’s possible to search within the currently selected project using the search field. Searches include strings, string IDs and comments.
```

Technically, you can search in All Projects, which is not a project. Not sure if my suggestion makes more sense.

Reply:
I think this suggestion works.